bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Tree.java

```diff
@@ -4347,7 +4348,11 @@ void setItemCount (int count, long hParent) {
 /*public*/ void setItemHeight (int itemHeight) {
 	checkWidget ();
 	if (itemHeight < -1) error (SWT.ERROR_INVALID_ARGUMENT);
-	OS.SendMessage (handle, OS.TVM_SETITEMHEIGHT, itemHeight, 0);
+	long newHeight = OS.SendMessage (handle, OS.TVM_SETITEMHEIGHT, itemHeight, 0);
```

Comment:
According to the [documentation of TVM_SETITEMHEIGHT](https://learn.microsoft.com/en-us/windows/win32/controls/tvm-setitemheight#return-value):
> # Return value
> Returns the previous height of the items, in pixels.

So this one should be renamed to `oldHeight`.

Reply:
I updated the branch. The solution is now much simpler as before. The behavior can just be reset by setting it to -1.